stories.json

```diff
@@ -31,7 +31,97 @@
+    "background": "northernlights"
+  },
+  {
+    "permalink": "fail-fast",
+    "text": "'If it hurts, do more of it' - I told my dad that it's a fail-fast environment and he told me I should fit right in ;D",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "unlimited-holidays",
+    "text": "Why we do not count holidays or work time? This goes back to the original founding of Giant Swarm, where we said that we need very good people and these people all do not work for money. Of course they need to be paid but they are all at a level where, on average, they do not move jobs for money, but rather for an interesting environment, good work topics, great people, and so forth. Self Actualisation if you will. They come to a company because they like the vision, the idea, and are intrinsically motivated to work there. As with most people, there are very few things I can do to closely monitor and control those people and it is already known that control is not a good motivator. Hence, my job as a “manager” needs to be to make you WANT to work. As a next step, if I manage to do that, and make sure to make it clear that everyone takes the rest they need, then I should just let you do that. Take rest, when and where you want. Especially in our current setting you might take your laptop to holidays and do a few hangouts from there, or read a business book on the weekend, play poker with friends and get one as a customer. The line between personal and work life blurs more and more and I want to support that, because it should and it should in a way that does not create stress. Quote: We all learned to answer our email on saturday morning but not to go to the cinema on tuesday at 14:00 - Ricardo Semler",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "honest-learning-environment",
+    "text": "From the beginning, everyone is super transparent about the steep learning curve involved in understanding and mastering your tasks. This honesty fosters an open-dialogue and a more productive learning environment.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "afk-for-everything",
+    "text": "AFK channel includes notes about family illness, going to the gym, taking a walk, and cooking lunch. To name just a few. People are encouraged to have a life outside of work.",
+    "background": "northernlights"
+  },
+  {
+    "permalink": "sharing-is-caring",
+    "text": "Communication can be formal or informal, it can be oral or written. I love our chat- insert topic people are interested (eg politics, music, pets, parenting, sports, book etc) channels. Why? Because people with less business touch points talk to each other, because more introverted people share, because you chose about what you want to talk <3 As a plus: We also have the channel #YELLING and #metalistening. For me this shows very good, what safe space Giant Swarm is for working.",
```

Comment:
```suggestion
    "text": "Communication can be formal or informal, it can be oral or written. I love our chat- insert topic people are interested (eg politics, music, pets, parenting, sports, book etc) channels. Why? Because people with less business touch points talk to each other, because more introverted people share, because you chose about what you want to talk <3 As a plus: We also have the channel #YELLING and #metalistening. For me this highlights what a safe space Giant Swarm is for working.",
```